operator.go

```diff
@@ -6,10 +6,21 @@ import (
 	"text/template"
 )
 
+type encoding string
```

Comment:
The point is that I don't use empty strings explicitly. I just want to have a `utf-8` value if I don't give any value to `Encoding` field at all.

But sure, consts are good, but I would like to keep String() method as well.